controllers/greptimedbcluster/greptimedbcluster_controller.go

```diff
@@ -89,6 +89,9 @@ func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
 // +kubebuilder:rbac:groups=greptime.io,resources=greptimedbclusters/finalizers,verbs=update
 // +kubebuilder:rbac:groups=core,resources=pods,verbs=get;list;watch;create;
 // +kubebuilder:rbac:groups=core,resources=services,verbs=get;list;watch;create;update;delete;
+// +kubebuilder:rbac:groups=core,resources=secrets,verbs=get;
+// +kubebuilder:rbac:groups=core,resources=configmaps,verbs=get;create;update;patch;delete;
+// +kubebuilder:rbac:groups=core,resources=events,verbs=get;list;watch;create;patch;
```

Comment:
suggestion：The configmap create policy is enough, may be not need event policy ?